docs/admin/setup/pyris.rst

```diff
@@ -31,70 +25,214 @@ E.g.: ``git clone https://github.com/ls1intum/Pyris.git Pyris``
+
+Example OpenAI
+**************
+.. code-block:: yaml
+
+    - api_key: <your_openai_api_key>
+      capabilities:
+        context_length: 16385
+        gpt_version_equivalent: 3.5
+        image_recognition: false
+        input_cost: 0.5
+        json_mode: true
+        output_cost: 1.5
+        privacy_compliance: false
+        self_hosted: false
+        vendor: OpenAI
+      description: GPT 3.5 16k
+      id: oai-gpt-35-turbo
+      model: gpt-3.5-turbo
+      name: GPT 3.5 Turbo
+      type: openai_chat
+
+Example Azure OpenAI
+********************
+.. code-block:: yaml
 
-- ``PYRIS_DOCKER_TAG``: The Docker tag to use. Ideally use ``latest`` here.
-- ``PYRIS_APPLICATION_YML_FILE``: The absolute path to the ``application.yml`` file that you created. E.g.: ``/opt/pyris/application.yml``
-- ``NGINX_PROXY_SSL_CERTIFICATE_PATH``: The absolute path to the SSL certificate to use. Defaults to self-signed certificates. For production, you should use a valid SSL certificate. For local development, you an ignore this variable.
-- ``NGINX_PROXY_SSL_CERTIFICATE_KEY_PATH``: The absolute path to the SSL certificate key to use. Defaults to self-signed certificates. For production, you should use a valid SSL certificate. For local development, you an ignore this variable.
+      api_key: "<your_azure_api_key>"
+      tools: []
+      capabilities:
+        input_cost: 6
+        output_cost: 16
+        gpt_version_equivalent: 4.5 # This is the equivalent GPT version of the model. We use 4.5 for GPT 4 Omni model.
+        context_length: 128000
+        vendor: "OpenAI"
+        privacy_compliance: True
+        self_hosted: False
+        image_recognition: True
+        json_mode: True
+      description: "GPT 4 Omni on Azure"
+      id: "azure-gpt-4-omni"
+      name: "GPT 4 Omni"
+      type: "azure_chat"
+      endpoint: <your_azure_model_endpoint>
+      api_version: "2024-02-15-preview"
+      azure_deployment: "gpt4o"
+      model: "gpt4o"
 
 Start Pyris
 """""""""""
+Using local environment
+***********************
+
+.. warning::
+    For local Weaviate vector database setup, please refer to `Weaviate Docs <https://weaviate.io/developers/weaviate/quickstart>`_.
+
+=============
+Prerequisites
+=============
+- Clone the Pyris repository to your local machine.
+- Ensure you correctly configured ``llm_config.local.yml`` file.
+- Ensure you correctly configured ``application.local.yml`` file.
+
+=======================================================
+Setup instructions
+=======================================================
+Follow the following steps for the local Pyris setup:
+
+1. **Check python version:** python --version (should be 3.12)
+2. **Install packages:** pip install -r requirements.txt
+3. **Start Pyris** using the following command:
+    .. code-block:: bash
+
+      APPLICATION_YML_PATH=<path-to-your-application-yml-file> LLM_CONFIG_PATH=<path-to-your-llm-config-yml> uvicorn app.main:app --reload
+
```

Comment:
I guess that's more regarding development?

Reply:
Yes this is more regarding the local development. I have added an indication regarding local development.